src/api/DocManagerApi.js

```diff
@@ -1,22 +1,30 @@
 import axios from "axios";
 
-const getCollectionDocuments = async collectionId => {
+export async function getCollectionDocuments(collectionId) {
   var path = `/collections/${collectionId}/documents`;
   console.log(
     "retrieving collection documents at " + process.env.REACT_APP_API_URL + path
   );
 
-  return await axios
-    .get(process.env.REACT_APP_API_URL + path)
-    .then(res => res.data);
-};
+  try {
+    return await axios
+      .get(process.env.REACT_APP_API_URL + path)
+      .then(res => res.data);
+  } catch (error) {
+    console.error(error);
+    throw error;
+  }
+}
 
-const getDocument = async documentId => {
+export async function getDocument(documentId) {
   var path = `/documents/${documentId}`;
   console.log("retrieving document at " + process.env.REACT_APP_API_URL + path);
-  return await axios
-    .get(process.env.REACT_APP_API_URL + path)
-    .then(res => res.data);
-};
-
-export { getCollectionDocuments, getDocument };
+  try {
+    return await axios
+      .get(process.env.REACT_APP_API_URL + path)
+      .then(res => res.data);
+  } catch (error) {
+    console.error(error);
+    throw error;
+  }
+}
```

Comment:
Added try/catch above to log the error (so we can't miss it), then throw it so caller can handle as desired.

Reply:
We did not wrap our api methods in try/catch blocks. We decided to delay any error handling until our Error Handling user story